components/bl-worldClock-component/component.json

```diff
@@ -0,0 +1,215 @@
+        },
+        {
+          "value": "-06:00",
+          "label": "GMT-6"
+        },
+        {
+          "value": "-05:00",
+          "label": "GMT-5"
+        },
+        {
+          "value": "-04:00",
+          "label": "GMT-4"
+        },
+        {
+          "value": "-03:00",
+          "label": "GMT-3"
+        },
+        {
+          "value": "-02:00",
+          "label": "GMT-2"
+        },
+        {
+          "value": "-01:00",
+          "label": "GMT-1"
+        },
+        {
+          "value": "+00:00",
+          "label": "GMT+0"
+        },
+        {
+          "value": "+01:00",
+          "label": "GMT+1"
+        },
+        {
+          "value": "+02:00",
+          "label": "GMT+2"
+        },
+        {
+          "value": "+03:00",
+          "label": "GMT+3"
+        },
+        {
+          "value": "+04:00",
+          "label": "GMT+4"
+        },
+        {
+          "value": "+05:00",
+          "label": "GMT+5"
+        },
+        {
+          "value": "+06:00",
+          "label": "GMT+6"
+        },
+        {
+          "value": "+07:00",
+          "label": "GMT+7"
+        },
+        {
+          "value": "+08:00",
+          "label": "GMT+8"
+        },
+        {
+          "value": "+09:00",
+          "label": "GMT+9"
+        },
+        {
+          "value": "+10:00",
+          "label": "GMT+10"
+        },
+        {
+          "value": "+11:00",
+          "label": "GMT+11"
+        },
+        {
+          "value": "+12:00",
+          "label": "GMT+12"
+        }
+      ],
+      "defaultValue": "+00:00",
+      "settingTooltip": "Choose a time zone for the world clock."
+    },
+    {
+      "type": "text",
+      "name": "label",
+      "label": "Label",
+      "showInSettings": true,
+      "hasLogicHandler": true,
+      "handlerId": "labelLogic",
+      "defaultValue": "London",
+      "handlerLabel": "Label Logic",
+      "dataBinding": true,
+      "settingTooltip": "The name of a location/city/etc to show in the component.",
+      "handlerDescription": "This is a handler for the logic to determine the name of a location/city/etc to display it in the component."
+    },
+    {
+      "type": "select",
+      "name": "timeFormat",
+      "label": "Time Format",
+      "showInSettings": true,
+      "defaultValue": "12",
+      "hasLogicHandler": false,
+      "handlerId": "timeFormatLogic",
+      "options": [
+        {
+          "value": "12",
+          "label": "12"
+        },
+        {
+          "value": "24",
+          "label": "24"
+        }
+      ],
+      "settingTooltip": "Choose a time format for the world clock(only for digital types)."
+    },
+    {
+      "type": "checkbox",
+      "name": "displaySeconds",
+      "label": "Display Seconds",
+      "showInSettings": true,
+      "defaultValue": true,
+      "hasLogicHandler": false,
+      "handlerId": "displaySecondsLogic",
+      "settingTooltip": "Select to display the seconds(second hand)."
```

Comment:
`Show the second hand when enabled.`

or

`Disable to hide second hand.`

Or any other variation without "Select"